pkgs/os-specific/linux/lkvm/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "lkvm";
+  version = "2020-05-19";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git";
+    rev = "b4fc4f605fc66a0942e88f37b3a4b18671e32b0c";
+    sha256 = "0kyb3rm98wzwgjs06crcgb39547rr4px4vap8ivm7yqnwbphr67f";
+  };
+
+  buildInputs = [ libbfd zlib libaio glibc glibc.static];
+
+  buildPhase = ''
+    make all
+  '';
+  installPhase = ''
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. Though, in this case, it is better to rely on the default `installPhase` and set `makeFlags = [ "prefix=${placeholder "out"}" ];`.